Web/coolprop/LowLevelAPI.rst

```diff
@@ -25,6 +25,46 @@ which yields the output:
 
 .. literalinclude:: snippets/AbstractState1.cxx.output
 
+Here is an example of the shared library usage with Julia wrapper::
+
+    julia> import CoolProp
+
+    julia> PT_INPUTS = CoolProp.get_input_pair_index("PT_INPUTS")
```

Comment:
Here is an example: https://github.com/CoolProp/CoolProp/commit/9c5a708f221a2ae154d68cc4b14656bd5741558b#diff-adfc138b4cc89b7e3ab2346946539625R70